app/services/users/services.rb

```diff
@@ -18,7 +18,7 @@ def initialize(user)
     #
     def authorizations
       @list << BackendServices::RX if user.authorize :mhv_prescriptions, :access?
-      @list << BackendServices::MESSAGING if user.authorize :mhv_messaging, :access?
+      @list << BackendServices::MESSAGING if user.authorize :legacy_mhv_messaging, :access?
```

Comment:
I think this is the only 'identity' owned code that was changed, and this appears to be a no-op for now. So I think from the 'identity' perspective this is good 2 go.

I already put any reservations about performance, etc, other comments, but it sounds like you all have a path forward for any potential issues there.